tests/unit/test_example_data.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+import pytest
+
+from ehrql.file_formats.csv import CSVRowsReader
+from ehrql.query_model.column_specs import get_column_specs_from_schema
+from ehrql.query_model.introspection import get_table_nodes
+from ehrql.tables import core, tpp
+
+
+filenames = [
+    "addresses.csv",
+    "clinical_events.csv",
+    "medications.csv",
+    "ons_deaths.csv",
+    "patients.csv",
+    "practice_registrations.csv",
+]
+
+ql_tables = [
+    tpp.addresses,
+    tpp.clinical_events,
+    tpp.medications,
+    core.ons_deaths,
+    core.patients,
+    tpp.practice_registrations,
+]
+
+
+@pytest.mark.parametrize("filename,ql_table", zip(filenames, ql_tables))
+def test_read_rows(filename, ql_table):
```

Comment:
Could you add a comment about what this test is doing? It's not immediately obvious that CSVRowsReader is validating it against the table's column specs